public/index.html

```diff
@@ -17,6 +17,7 @@
   <link rel="icon" type="image/png" href="%PUBLIC_URL%/favicon-16x16.png" sizes="16x16" lazyload>
   <link rel="mask-icon" href="%PUBLIC_URL%/safari-pinned-tab.svg" color="#5bbad5" lazyload>
   <link rel="stylesheet" href="%PUBLIC_URL%/chart.css" lazyload>
+    <link rel="stylesheet" href="%PUBLIC_URL%/icons-font.css" lazyload>
```

Comment:
do we need this?

Reply:
@optimusway Absolutely! This is our main icons styles if we remove this link all icons disappear